### PR TITLE
Various funnel breakdown fixes

### LINKED ISSUE #6109: Funnels UI minor improvements
## Is your feature request related to a problem?

While using funnels I noticed a few things to improve on the new layout.

## Describe the solution you'd like

Edit: I'll address this in a separate issue.
- [ ] The tooltip is hard to read as there's a ton of stuff going on, maybe we can add some icons, a bit more spacing and then maybe things on parenthesis could have a smaller font and muted color. CC @clarkus for thoughts.

Edit: Below addressed in #6133

- [x] The checkbox has a foreign blue hue, ideally we would want to match the series color.
- [x] We've changed the main bar color from the main primary color but I think this helped make more obvious that you can click on the bar.
- [x] We use "Comp. rate" & "rate" interchangeably in the table header.
- [x] The average time can get into 2 lines in some cases which doesn't look great.
- [x] The tooltip's title shows a "None" (probably a breakdown value) but it doesn't make sense in this context.
- [x] The "Baseline" label can be confusing if you don't have a breakdown. 

## Describe alternatives you've considered

Not applicable.

## Additional context

CC @alexkim205

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Fixes #6109

- [x] The checkbox has a foreign blue hue, ideally we would want to match the series color.
- [x] We've changed the main bar color from the main primary color but I think this helped make more obvious that you can click on the bar.
- [x] We use "Comp. rate" & "rate" interchangeably in the table header.
- [x] The average time can get into 2 lines in some cases which doesn't look great.
- [x] The tooltip's title shows a "None" (probably a breakdown value) but it doesn't make sense in this context.
- [x] The "Baseline" label can be confusing if you don't have a breakdown.   

Screen Shot 2021-09-27 at 2 07 03 PM

Screen Shot 2021-09-27 at 2 10 30 PM

## How did you test this code?

Stylistic improvements don't require extensive testing.

### CODE REVIEW

File: frontend/src/scenes/funnels/FunnelBarGraph.tsx [hunk 1]
```diff
@@ -97,7 +99,8 @@ export function BreakdownVerticalBarGroup({
                 const currentBarHeight = (height * breakdown.count) / basisBreakdownCount
                 const previousBarHeight =
                     (height * (previousStep?.nested_breakdown?.[breakdownIndex]?.count ?? 0)) / basisBreakdownCount
-                const color = getSeriesColor(breakdown.order) as string
+                const color = getSeriesColor(breakdown.order, isSingleSeries) as string
```
Comment (mariusandra) [inline frontend/src/scenes/funnels/FunnelBarGraph.tsx:102]: It's not new code, but this type override seems like a potential source of bugs.

File: frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx [hunk 2]
```diff
@@ -49,6 +49,9 @@ export function FunnelStepTable({ filters: _filters, dashboardItemId }: Omit<Cha
         if (isNewVertical) {
             const _columns: ColumnsType<FlattenedFunnelStepByBreakdown> = []
             const useCustomName = !!featureFlags[FEATURE_FLAGS.RENAME_FILTERS]
+            const isOnlySeries = flattenedBreakdowns.length === 1
+
+            console.log('breakdown', isOnlySeries)
```
Comment (mariusandra) [inline frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx:54]: log!

**Review by mariusandra — APPROVED** [review]
LGTM, save for one console.log and one shady type.

**Review by clarkus — APPROVED** [review]

**Comment by paolodamico** [comment]
Love this @alexkim205, thanks!

**Pull request merged**